recipes/antic/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+chmod +x configure
+
+export CFLAGS="$CFLAGS -funroll-loops -g -Wno-unknown-pragmas"
```

Comment:
What's the reason for the custom flags here?

Reply:
I copied them over from the FLINT recipe. But I guess we should try without first.
Thanks for the comment.

Reply:
flint/antic's configure overrides these default flags if CFLAGS is set in an env variable. So, these default flags need to be added back in.

Reply:
Ah great. Let me revert that change then.